src/qt/forms/debugwindow.ui

```diff
@@ -1074,8 +1087,8 @@
                <rect>
                 <x>0</x>
                 <y>0</y>
-                <width>300</width>
-                <height>426</height>
+                <width>274</width>
```

Comment:
why are you including changes to tab_peers in this? While the peers tab should be redesigned to maximize space, I don't think this PR (focused on the network traffic graph) should carry any changes to tab_peers